auth-web/src/components/auth/account-freeze/AccountOverviewNSF.vue

```diff
@@ -87,33 +93,39 @@
 import { computed, defineComponent, onMounted, reactive, toRefs } from '@vue/composition-api'
 import CommonUtils from '@/util/common-util'
 import { FailedInvoice } from '@/models/invoice'
-import moment from 'moment'
+import { useCodesStore } from '@/stores/codes'
 import { useOrgStore } from '@/stores/org'
 
 export default defineComponent({
   name: 'AccountOverviewNSFView',
   emits: ['step-forward'],
   setup (_, { emit }) {
+    const { getCodes } = useCodesStore()
     const orgStore = useOrgStore()
     const currentOrganization = computed(() => orgStore.currentOrganization)
     const currentMembership = computed(() => orgStore.currentMembership)
     const calculateFailedInvoices: any = orgStore.calculateFailedInvoices
     const downloadNSFInvoicesPDF: any = orgStore.downloadNSFInvoicesPDF
-    const formatDate = CommonUtils.formatDisplayDate
-    const suspendedDate = (currentOrganization.value?.suspendedOn) ? formatDate(moment(currentOrganization.value.suspendedOn)) : ''
 
     const state = reactive({
       nsfAmount: 0,
       totalAmount: 0,
       totalAmountRemaining: 0,
-      totalPaidAmount: 0
+      totalPaidAmount: 0,
+      suspensionReason: '',
+      suspendedDate: currentOrganization.value?.suspendedOn
+        ? CommonUtils.formatDateToHumanReadable(currentOrganization.value.suspendedOn) : ''
     })
 
     const goNext = () => {
       emit('step-forward')
     }
 
     onMounted(async () => {
+      const suspensionReasonCodes = await getCodes()
+      state.suspensionReason = suspensionReasonCodes.find(
+        suspensionReasonCode => suspensionReasonCode?.code === currentOrganization.value?.suspensionReasonCode
```

Comment:
I don't think this is right, you need to grab it from the non_sufficient_funds table?

Reply:
currentOrganization.suspensionReason != NSF reason

Reply:
![image](https://github.com/user-attachments/assets/65c194e0-8f68-424f-bbe4-fbf5ca6ad47e)

Reply:
I think you want this one:
![image](https://github.com/user-attachments/assets/d6d1de6f-b04d-473f-95d1-1fb8c860b1e2)